excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/endpoint/exception/SpreadsheetValidationErrorMessageCreatorTest.java

```diff
@@ -535,29 +535,29 @@ public void caseFieldEntityInvalidUserRole_createErrorMessageCalled_customMessag
     }
 
     @Test
-    public void caseFieldUserRoleEntityInvalidUserRole_createErrorMessageCalled_defaultMessageReturned() {
+    public void caseFieldEntityInvalidUserRole_createErrorMessageCalled_defaultMessageReturned() {
```

Comment:
Rename test when validation error `CaseFieldEntityInvalidUserRoleValidationError` is renamed

Reply:
Fixed in 3951e2b